assets/html/favorites.html

```diff
@@ -59,11 +59,11 @@ <h1>Distract Me</h1>
 		</header>
 		<h2>Favorites</h2>
 		<div class="container row">
-			<div class="favorite-column col s6">
+			<div class="favorite-column col s5">
 				<h3><i class="material-icons">tag_faces</i> Jokes</h3>
 				<div class="fav-jokes" id="favorites">jokes</div>
 			</div>
-			<div class="favorite-column col s6">
+			<div class="favorite-column col s5">
```

Comment:
in the future.. these must be divisible by 12 in order to accurately be responsive